src/scf/self_consistent_field.jl

```diff
@@ -29,14 +29,17 @@ function next_density(ham::Hamiltonian;
 end
 
 function scf_default_callback(info)
-    E = info.energies === nothing ? Inf : sum(values(info.energies))
-    res = norm(info.ρout.fourier - info.ρin.fourier)
     if info.n_iter == 1
-        label = haskey(info.energies, "Entropy") ? "Free energy" : "Energy"
-        @printf "Iter   %-15s    ρout-ρin\n" label
-        @printf "----   %-15s    --------\n" "-"^length(label)
+        E_label = haskey(info.energies, "Entropy") ? "Free energy" : "Energy"
+        @printf "n     %-12s      ρout-ρin   Eₙ₋₁-Eₙ     diag\n" E_label
```

Comment:
`diag` or `diag #it`? No strong opinion here.

Reply:
Also diag is off by one space and for some reason I would indent the "Energy" label by one space (to not align with the - but the number) ... of course debatable. Also the capitalisation of the headers should be consistent (I'd just lower-case energy and entropy)